src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -1561,6 +1561,14 @@ let internalFlags (tcConfigB: TcConfigBuilder) =
             None
         )
 
+        CompilerOption(
+            "tokenize-debug",
+            tagNone,
+            OptionUnit(fun () -> tcConfigB.tokenize <- TokenizeOption.Debug),
+            Some(InternalCommandLineOption("--tokenize-debug", rangeCmdArgs)),
+            None
```

Comment:
Consider adding some help text here, would be useful for future generations :)

Reply:
I'd rather prefer either help texts for more internal options or none (as it currently stands). Adding a text for a single option doesn't look much better to me. 🙂

Reply:
Well, if you wish you can add help texts everywhere you feel it would help :D 

I leave it on you - if you think of smth, feel free to add it, otherwise feel free to resolve.

Reply:
I think I'll mark it resolved for now, I'm just trying to make it easier and faster to work on the LexFilter changes :)